examples/dct_hash/dct_hash.cc

```diff
@@ -0,0 +1,29 @@
+#include "imgproc/dct_hash.h"
+#include <opencv2/core.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/highgui.hpp>
+
+int bobMain(int, char **)
+{
+	std::string image_path1 = "library1.jpg";
+	std::string image_path2 = "library2.jpg";
+    cv::Mat img1 = imread(image_path1, cv::IMREAD_GRAYSCALE);
+	cv::Mat img2 = imread(image_path2, cv::IMREAD_GRAYSCALE);
```

Comment:
Another tabs/spaces issue.

VSCode can automatically sort this out. If you install the editorconfig extension then your whitespace will be sorted out when you save a file.